### PR TITLE
vmui: increase font-size and fix the text display

### PR DESCRIPTION
**Pull request by Loori-R**
- set default font size to `14px`
- fix the text display on buttons in Safari 16.4

### CODE REVIEW

**Review by dmitryk-dk — COMMENTED** [review]
Overall looks good! One small comment

File: app/vmui/packages/vmui/src/components/Main/Select/style.scss [hunk 1]
```diff
@@ -5,21 +5,20 @@
...
     &-content {
       display: flex;
       align-items: center;
       justify-content: flex-start;
       flex-wrap: wrap;
       gap: $padding-small;
-      width: 100%;
-      max-width: calc(100% - ($padding-global + 61px));
+      flex-grow: 1;
+      //max-width: calc(100% - ($padding-global + 61px));
```
[the remark this answers is not in the document]
Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Main/Select/style.scss]: updated

**Review by dmitryk-dk — APPROVED** [review]

**Comment by hagen1778** [comment]
@k1rk @Amper could you please try building this VMUI version to verify the issue with button labels is gone ?

**Comment by Amper** [comment]
> @k1rk @Amper could you please try building this VMUI version to verify the issue with button labels is gone ?

Yes, everything is ok!

image

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): properly return empty slices instead of nil for `/api/v1/rules` and `/api/v1/alerts` API handlers. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4221).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix the display of the tenant selector. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4160).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix UI freeze when the query returns non-histogram series alongside histogram series.
+* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix the text display on buttons in Safari 16.4.
```
Comment (hagen1778) [inline docs/CHANGELOG.md:65]: please mention font size increase as well

Comment (Loori-R) [reply docs/CHANGELOG.md:69]: Added to the FEATURE section:
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4273/files#diff-2bfab3db5cc1baf4c6d3ff6b19901926e3bdf4411ec685dac973e5fcff1c723bR47

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): display histograms as heatmaps in [Metrics explorer](https://docs.victoriametrics.com/#metrics-explorer). See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4111).
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add `WITH template` playground. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3811).
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add ability to debug relabeling. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3807).
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): increase default font size to `14px`.
```
Comment (hagen1778) [inline docs/CHANGELOG.md:47]:
```suggestion
* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): increase default font size to `14px` for better usability.
```

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**